lib/okta.go

```diff
@@ -483,6 +483,7 @@ type OktaProvider struct {
 	SessionDuration time.Duration
 	OktaAwsSAMLUrl  string
 	MFADevice       string
+	OktaSessionKey  string
```

Comment:
Could this have a more descriptive name? It's the key for the keychain item containing the Okta session cookie. How about `SessionCookieKey`? Even that doesn't sound descriptive enough. Maybe add a comment above to flesh it out.